src/Service/PdfCreator.php

```diff
@@ -4,38 +4,36 @@
 
 namespace App\Service;
 
-use Spipu\Html2Pdf\Html2Pdf;
+use Dompdf\Dompdf;
+use Dompdf\Options;
 
 final class PdfCreator implements PdfCreatorInterface
 {
     private $pdf;
     private $orientation;
     private $format;
-    private $lang;
-    private $unicode;
-    private $encoding;
-    private $margin;
 
-    public function __construct(string $orientation, string $format, string $lang, bool $unicode, string $encoding, array $margin)
+    public function __construct(string $orientation, string $format)
     {
         $this->orientation = $orientation;
         $this->format = $format;
-        $this->lang = $lang;
-        $this->unicode = $unicode;
-        $this->encoding = $encoding;
-        $this->margin = $margin;
     }
 
     public function create(): void
     {
-        $this->pdf = new Html2Pdf($this->orientation, $this->format, $this->lang, $this->unicode, $this->encoding, $this->margin);
-        $this->pdf->setTestIsImage(false);
+        $this->pdf = new Dompdf(new Options([
+            'isHtml5ParserEnabled' => true,
+            'isRemoteEnabled' => true,
+        ]));
+        $this->pdf->setPaper($this->format, $this->orientation);
     }
 
-    public function generatePdf(string $template, string $name): string
+    public function generatePdf(string $template, string $name): void
```

Comment:
Why not still returning the generated name? In the controller GeneratePdf you have 
`return $this->pdfCreator->generatePdf($template, 'brochure-community-event');` in the handle that return the generated pdf name, so you need to stick with returning the pdf name

Reply:
Or change the Handle in the GeneratePdf Controller :)